cli/src/commands/apps/updateApp.ts

```diff
@@ -0,0 +1,57 @@
+import { flags } from '@oclif/command'
+import chalk from 'chalk'
+import { IAppMetadata, IMemberRemarked, MemberRemarked } from '@joystream/metadata-protobuf'
+import { metadataToBytes } from '../../helpers/serialization'
+import { getInputJson } from '../../helpers/InputOutput'
+import AppCommandBase from '../../base/AppCommandBase'
+import MembershipsCommandBase from '../../base/MembershipsCommandBase'
+
+export default class GenerateAppCreationMessage extends AppCommandBase {
+  static description = 'App creation message factory'
+
+  static flags = {
+    input: flags.string({
+      required: false,
+      char: 'i',
+      description: `Path to JSON file containing app details`,
+    }),
+    appId: flags.string({
+      required: true,
+      description: `ID of the app to update`,
+    }),
+    skip: flags.boolean({
+      char: 's',
+      description: "If true command won't prompt missing fields",
+    }),
```

Comment:
Actually here I don't think `skip` makes any sense without `input`, so you can add a dependency:
```suggestion
    skip: flags.boolean({
      char: 's',
      description: "If true command won't prompt missing fields",
      dependsOn: ['input'],
    }),
```